contrib/st2/opensds/actions/attach_volume.py

```diff
@@ -20,30 +20,30 @@
 
 class AttachVolumeAction(Action):
     def run(self,
-            ip_addr="",
-            port="",
-            tenant_id="",
+            osds_ip="",
```

Comment:
Attn!!!No need to differentiate these with prefix. From API perspective, we give the API information along with parameters needed (type, required/not, description). No need to differentiate in any other ways. The user of the API need to decide how to get the param to send to the API. Please revert these prefix changes for all such changes.